ibis/pandas/udf.py

```diff
@@ -300,51 +334,52 @@ def execute_udf_node_groupby(op, *args, **kwargs):
         @functools.wraps(func)
         def wrapped(*args):
             return UDFNode(*args).to_expr()
-
         return wrapped
-
     return wrapper
 
 
-def udaf(input_type, output_type):
+def udaf(input_type, output_type, analytic=False):
     """Define a UDAF (user-defined aggregation function) that takes N pandas.
     Series or scalar values as inputs.
 
     Parameters
     ----------
-    input_type : List[T]
-        A rules encoding the abstract type of each argument or one of the types
-        found in :mod:`~ibis.expr.datatypes`. These are found in
-        :mod:`~ibis.expr.rules`. The length of this list must match the number
-        of arguments to the function. Argument splatting in the UDF signature
-        is not supported.
-    output_type : rule or ibis.expr.datatypes.DataType
-        The abstract return type of the function. This *cannot* be a rule that
-        encodes a :class:`~ibis.expr.types.ColumnExpr` since this API is for
-        defining *aggregation* functions.
+    input_type : List[ibis.expr.datatypes.DataType]
+        A list of the types found in :mod:`~ibis.expr.datatypes`. The length of
+        this list must match the number of arguments to the function. Variadic
+        arguments are not yet supported.
+    output_type : Union[ibis.expr.datatypes.DataType]
+        The return type of the function.
+    analytic : bool
+        If ``True`` this function will is sensitive group by operations.
```

Comment:
I don't totally understand this this doc.

Reply:
This is gone now.